kalman_filter.q

```diff
@@ -0,0 +1,42 @@
+// Reference
+// https://www.quantstart.com/articles/State-Space-Models-and-the-Kalman-Filter/
+
+// Auxiliary functions
+// Creates a diagonal matrix
+eye:{(2#x)#1f,x#0f}

```

Comment:
Consider this shorter version :) 
```suggestion
eye:{{x=/:x}til x}

```

Reply:
I can try that but I dont know if it is going to cause some problems as we need floats and not booleans or integers

Reply:
It shouldn't cause problems, try e.g.: `4f*eye 4`

Reply:
This implementation causes problems, I think it has to do with types